### PR TITLE
Add DashPvP

### PR DESCRIPTION
**Pull request by andrewbelling**

### CODE REVIEW

File: servers.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
         "id": "dashpvp",
         "name": "DashPvP",
         "addresses": [
-            "play.dashpvp.net"
+            "play.dashpvp.net",
```
Comment (GrowlyX) [inline servers.json:6]: Lunar Client services automatically detect sub-domains, please remove this.

**Review by GrowlyX — CHANGES_REQUESTED** [review]

File: servers.json [hunk 2]
```diff
@@ -3,8 +3,7 @@
         "id": "dashpvp",
         "name": "DashPvP",
         "addresses": [
-            "play.dashpvp.net",
-            "dashpvp.net"
+            "play.dashpvp.net"
```
Comment (GrowlyX) [inline servers.json:6]: No, keep dashpvp.net, remove `play.dashpvp.net`.

**Review by GrowlyX — CHANGES_REQUESTED** [review]

File: servers.json (hunk 2 above)
Comment (OmenDoesStuff) [reply servers.json:6]: dashpvp.net isnt a usable IP


> No, keep dashpvp.net, remove `play.dashpvp.net`.

Comment (GrowlyX) [reply servers.json:6]: I know, but Lunar Client already checks for sub-domains, you don't need to specify it here.

Comment (OmenDoesStuff) [reply servers.json]: > I know, but Lunar Client already checks for sub-domains, you don't need to specify it here.

Done

**Review by GrowlyX — APPROVED** [review]

File: servers.json (hunk 2 above)
Comment (GrowlyX) [reply servers.json]: Thank you!

**Pull request merged**